pom.xml

```diff
@@ -314,6 +315,28 @@
       <version>2.1.3</version>
       <scope>test</scope>
     </dependency>
+    <!--
+    testcontainers
+    keycloak testcontainer
+    libraries needed for keycloak client
+    -->
+    <dependency>
+      <groupId>com.github.dasniko</groupId>
+      <artifactId>testcontainers-keycloak</artifactId>
+      <version>3.4.0</version>
+      <scope>test</scope>
+    </dependency>
+    <dependency>
+      <groupId>jakarta.annotation</groupId>
+      <artifactId>jakarta.annotation-api</artifactId>
+      <version>2.1.1</version>
+      <scope>test</scope>
+    </dependency>
+    <dependency>
+      <groupId>jakarta.xml.bind</groupId>
+      <artifactId>jakarta.xml.bind-api</artifactId>
+      <version>3.0.1</version>
+    </dependency>
```

Comment:
Why the need for these dependencies? I don't see us consuming them directly anywhere in this PR. If this is to resolve some upper bounds error, should this be placed in `<dependencyManagement>` instead?

Reply:
those are dependency of the keycloak testcontainer, more precisely the keycloak admin client. However they have scope provided and they weren't found in the local execution. After your comment I've double checked to post here the stacktrace ... but it's working now in local ( :confused: ). I've removed then and let's check with the CI if they are really needed or not

Reply:
Thanks, I agree these are still needed. But it wasn't obvious as to why, and I (or someone else) might forget about this conversation and be tempted to remove them in the future, so I think it would be good to add a comment stating what you just said above (in particular, that these are provided dependencies of the Keyclock admin client). That way, their purpose remains clear to future readers.